projects/packages/forms/src/dashboard/inbox/index.js

```diff
@@ -15,30 +15,33 @@ import InboxList from './list';
 import InboxResponse from './response';
 import './style.scss';
 
-const RESPONSES_FETCH_LIMIT = 20;
+const RESPONSES_FETCH_LIMIT = 5;
```

Comment:
Hmm, is that really something we want or is this just for testing?

Reply:
yeah, that was merely for testing, must have slipped in